compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -140,7 +140,8 @@ struct ConvertToROCDLPass : public ConvertToROCDLBase<ConvertToROCDLPass> {
       RewritePatternSet patterns(&getContext());
       // These patterns only convert a subset of arith that target specific
       // rocdl intrinsics (e.g. fp8 conversions).
-      arith::populateArithToAMDGPUConversionPatterns(patterns);
+      arith::populateArithToAMDGPUConversionPatterns(
+          patterns, /*saturateFP8Truncf=*/false);
```

Comment:
@kuhar My fix is making it be false, because I found that the [default value](https://github.com/llvm/llvm-project/commit/750e90e4403df23d6b271afb90e6b4d463739965#diff-885a493f2c17594b2ae619274be52b288adbd4ced4ac254f26c5502f82617221R128-R133) is false. Does it make sense to you?

Reply:
Yes. And we don't rely on trubcsting fp8 anywhere yet.